.coveragerc

```diff
@@ -12,4 +12,7 @@ omit =
     #src/RestrictedPython/tests/*.py
 
 [report]
-precision = 3
+precision = 2
+
+[html]
+directory = reports/coverage
```

Comment:
well these lines are about coverage reports not flake8 reports. 
The config just moves them into a directory reports.